src/mbgl/gl/context.hpp

```diff
@@ -144,6 +144,8 @@ class Context : private util::noncopyable {
         return { size, createTexture(size, nullptr, format, unit) };
     }
 
+    UniqueTexture createTexture(Size size, const void* data, TextureFormat, TextureUnit);
```

Comment:
I'd prefer if we kept this method private and use a more high level `createTexture` call instead.